runbotics/runbotics-orchestrator-ui/src/components/InfoPanel/ProcessInstanceDetails/ProcessInstanceDetailsHeader.tsx

```diff
@@ -18,7 +19,10 @@ const isProcessInstanceActive = (
 ) => status === ProcessInstanceStatus.INITIALIZING
     || status === ProcessInstanceStatus.IN_PROGRESS;
 
-const ProcessInstanceDetailsHeader: VFC<Props> = ({ processInstance }) => (
+const ProcessInstanceDetailsHeader: VFC<Props> = ({ processInstance }) => {
+    const convertedKey = convertToPascalCase(processInstance.status)
```

Comment:
names of constants should explain what is the content of it. Rename it to e.g. `formattedStatus`